libs/membership/api/setup-database.js

```diff
@@ -0,0 +1,9 @@
+import execa from 'execa'
+import {name} from './jest.config'
+
+export default async () => {
+  const databaseUrl = `postgresql://postgres@localhost:5432/${name}?schema=public`
+  process.env.DATABASE_URL = databaseUrl
```

Comment:
Shouldn't this be take from the process.env ? :o

Reply:
This is a test database :D